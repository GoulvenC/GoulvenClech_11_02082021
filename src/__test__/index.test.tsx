import React from 'react'
import renderer from 'react-test-renderer'
import {cleanup, fireEvent, render} from '@testing-library/react';
import HousingSection from "../components/index/housing-section"

describe("Given I am on the index page", () => {
    test("Then the housing section should be rendered", () => {
        const component = renderer.create(
            <HousingSection />
        )
        let tree = component.toJSON()
        expect(tree).toMatchSnapshot()
    }) 
    test("Second test", () => {
        const {queryByLabelText} = render(
            <HousingSection />
        )
        console.log(queryByLabelText(/cosy/i))
    })    
})
