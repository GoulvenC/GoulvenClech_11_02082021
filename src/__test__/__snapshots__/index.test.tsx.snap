// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Given I am on the index page Then the housing section should be rendered 1`] = `
<section
  className="housing-section my-4 p-8 rounded-3xl bg-gray-100"
>
  <span
    className="loading-message"
  >
    Loading...
  </span>
  <span
    className="error-message"
  />
  <div
    className="housing-wrapper grid lg:grid-cols-3 md:grid-cols-2 grid-cols-1 gap-8"
  />
</section>
`;
